linkerd/app/gateway/src/lib.rs

```diff
@@ -153,7 +153,7 @@ mod test {
                 dst: dst_name
                     .map(|n| NameAddr::from_str(n).unwrap().into())
                     .unwrap_or_else(|| socket_addr.into()),
-                http_settings: http::Settings::Http2,
+                http_version: http::Version::Http1,
```

Comment:
Is this now `Http1` because we know it should be upgraded to `H2`? If not, why this change?

Reply:
it's a test, so it's not actually used